### ISSUE TITLE
Route constraints are not stripped and generate invalid clients

### ISSUE THREAD

**Issue by cschleiden** [NSwag/NSwag#214]
A controller action like this:
```
[Route("{id:long:min(1)}")]
public IHttpActionResult Get(long id)
{
  //...
}
```

results in a `swagger.json` like this:

```
"/api/resource/{id:long:min(1)}": {
      "get": {
        "tags": [
          "MyController"
        ],
[...]
        "parameters": [
          {
            "type": "integer",
            "name": "id",
            "in": "path",
            "required": true,
            "x-nullable": false,
            "format": "long"
          }
        ],
```

which in turns generates a client that contains code like the following:

```
public async Task<bye[]> GetAsync(long id, CancellationToken cancellationToken)
{
  var url_ = string.Format("{0}/{1}?", BaseUrl, "api/resource/{id:long:min(1)}");
  if (id == null)
    throw new ArgumentNullException("id");
    
  url_ = url_.Replace("{id}", Uri.EscapeUriString(id.ToString()));
```

which does not work, because the route constraint is included and `{id}` doesn't match anything. 

Is there a way to strip the route constraints and only keep the parameter name?

**Comment by rsuter** [comment]
This is a bug and will be fixed soon... You can loop through all operations and fix this manually...

**Comment by rsuter** [comment]
https://github.com/NSwag/NSwag/commit/f65fc8230c3c7cb6167f3f24bbbc5bf95eae3be5

**Comment by cschleiden** [comment]
Perfect, thanks a lot for the fast response.

**Comment by rsuter** [comment]
Available in v3.28
